### DISCUSSION TITLE
Funding.yml

### DISCUSSION THREAD

**Discussion by jekkos** [opensourcepos/opensourcepos#3682] [Team Posts]
As some of you might have noticed, I have added a FUNDING.yml to the project. I think this is a easy way for contributors want to get something in return from the community. 

Feel free to add your own profile to the file so people can choose who to reward in case credit is due. 
Personally I'd lke to get a little bit out of it as I'd like to make the project self sustainable (in this case just cover the monthly DO expense, which for this month is covered by a referral payout, but these are becoming less frequent for sure).

**Comment by objecttothis** [comment]
Good idea.  As for me, the work I contribute is almost universally to benefit my use case.  In other words, I would do it anyway.  I'm happy for anyone wanting to pay it back to send that to you so that expenses get covered.  I wonder if it could be beneficial to have a community edition of the software that has a single small google ads banner as a way to help cover costs?  That way it's unobtrusive for users, but could still generate even something small.  Then you could also maintain an ad free version that is provided at a per major or minor release level for a small cost.  There would be some design and dev considerations, but it could help cover costs.

**Comment by jekkos** [comment]
I think that most of the time I'm able to cover the costs for a month, in 2020 it happened maybe that there were no referral payouts for 3 months. The other months were covererd by the referral link which is nice. 
Basically I think it's fine if it continues like this, I don't mind having to pay a bit extra so now and then. Also I'm lucky that the euro is so much stronger than dollars now and so this service has become about 10% cheaper for me..
